omise.js/form-example/app.js

```diff
@@ -4,28 +4,29 @@
 
 Omise.setPublicKey('YOUR_PUBLIC_KEY');
 
-var cardForm     = document.getElementById('card-form');
 var checkoutForm = document.getElementById('checkout-form')
 
-cardForm.addEventListener('submit', submitHandler, false);
+checkoutForm.addEventListener('submit', submitHandler, false);
 
 // Submit handler for checkout form.
 function submitHandler(event) {
   event.preventDefault();
 
+  // NOTE: Using `data-name` because we don't want to send any credit card information to the server.
```

Comment:
@ratchagarn 

for note, how about
```html
<!-- 
NOTE: Using `data-name` to prevent sending credit card information fields to the backend server via HTTP Post
(according to the security best practice https://www.omise.co/security-best-practices#never-send-card-data-through-your-servers).
-->
```